graphql/queryResolvers/moduleCrud.ts

```diff
@@ -0,0 +1,29 @@
+import type { MutationDeleteModuleArgs, MutationAddModuleArgs } from '..'
+import prisma from '../../prisma'
+
+export const modules = async () => {
+  return prisma.module.findMany({
+    include: {
+      author: true
+    }
+  })
+}
+
+export const addModule = async (_parent: void, args: MutationAddModuleArgs) => {
+  const { authorId, content, lessonId, name } = args
+  if (!authorId || !content || !name || !lessonId) {
+    throw new Error('Missing parameters')
+  }
+  return await prisma.module.create({
+    data: { authorId, content, lessonId, name }
+  })
+}
+
+export const deleteModule = async (
+  _parent: void,
+  args: MutationDeleteModuleArgs
+) => {
+  const { id } = args
+  await prisma.module.delete({ where: { id } })
```

Comment:
We should wrap this in a try/catch block

Reply:
what do you suggest we do in the `catch` block @anthonykhoa ?

Reply:
Log the error. Incase anything goes wrong, logs would be helpful in issue triaging.

Reply:
@mino323 See comment thread on the `createModule` function for the discussion about error handling